addons/crm_claim/crm_claim.py

```diff
@@ -38,144 +18,102 @@ class crm_claim_stage(osv.osv):
-
-    _columns = {
-        'id': fields.integer('ID', readonly=True),
-        'name': fields.char('Claim Subject', required=True),
-        'active': fields.boolean('Active'),
-        'action_next': fields.char('Next Action'),
-        'date_action_next': fields.datetime('Next Action Date'),
-        'description': fields.text('Description'),
-        'resolution': fields.text('Resolution'),
-        'create_date': fields.datetime('Creation Date' , readonly=True),
-        'write_date': fields.datetime('Update Date' , readonly=True),
-        'date_deadline': fields.date('Deadline'),
-        'date_closed': fields.datetime('Closed', readonly=True),
-        'date': fields.datetime('Claim Date', select=True),
-        'ref': fields.reference('Reference', selection=openerp.addons.base.res.res_request.referencable_models),
-        'categ_id': fields.many2one('crm.claim.category', 'Category'),
-        'priority': fields.selection([('0','Low'), ('1','Normal'), ('2','High')], 'Priority'),
-        'type_action': fields.selection([('correction','Corrective Action'),('prevention','Preventive Action')], 'Action Type'),
-        'user_id': fields.many2one('res.users', 'Responsible', track_visibility='always'),
-        'user_fault': fields.char('Trouble Responsible'),
-        'team_id': fields.many2one('crm.team', 'Sales Team', oldname='section_id',\
-                        select=True, help="Responsible sales team."\
-                                " Define Responsible user and Email account for"\
-                                " mail gateway."),
-        'company_id': fields.many2one('res.company', 'Company'),
-        'partner_id': fields.many2one('res.partner', 'Partner'),
-        'email_cc': fields.text('Watchers Emails', size=252, help="These email addresses will be added to the CC field of all inbound and outbound emails for this record before being sent. Separate multiple email addresses with a comma"),
-        'email_from': fields.char('Email', size=128, help="Destination email for email gateway."),
-        'partner_phone': fields.char('Phone'),
-        'stage_id': fields.many2one ('crm.claim.stage', 'Stage', track_visibility='onchange',
-                domain="['|', ('team_ids', '=', team_id), ('case_default', '=', True)]"),
-        'cause': fields.text('Root Cause'),
-    }
-
-    _defaults = {
-        'user_id': lambda s, cr, uid, c: uid,
-        'team_id': lambda s, cr, uid, c: s._get_default_team_id(cr, uid, c),
-        'date': fields.datetime.now,
-        'company_id': lambda s, cr, uid, c: s.pool.get('res.company')._company_default_get(cr, uid, 'crm.case', context=c),
-        'priority': '1',
-        'active': lambda *a: 1,
-        'stage_id': lambda s, cr, uid, c: s._get_default_stage_id(cr, uid, c)
-    }
-
-    def stage_find(self, cr, uid, cases, team_id, domain=[], order='sequence', context=None):
+    _inherit = 'mail.thread'
+
```

Comment:
Despite using List it is inheriting only one Model. List is used for inheriting multiple Models.